src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
+    private String peeling;
+    private String juice;
+    private String color;
+
+    public Apple(String stem, String blossom, String peeling, String juice, String color) {
+        this.stem = stem;
+        this.blossom = blossom;
+        this.peeling = peeling;
+        this.juice = juice;
+        this.color = color;
+    }
+
+    public String getPeeling() {
+        return peeling;
+    }
+
+    public void setPeeling(String peeling) {
+        this.peeling = peeling;
+    }
+
+    public String getJuice() {
+        return juice;
+    }
+
+    public void setJuice(String juice) {
+        this.juice = juice;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getSize() {
+        return size;
+    }
+
+    public void setSize(String size) {
+        this.size = size;
+    }
+
+    public String getTexture() {
+        return texture;
+    }
+
+    public void setTexture(String texture) {
+        this.texture = texture;
+    }
+
+    public String getDamage() {
+        return damage;
+    }
+
+    public void setDamage(String damage) {
+        this.damage = damage;
+    }
+
+    private String size;
+    private String texture;
```

Comment:
What is an example content for this String? Would it be wrinkly? Smooth? Other? What purpose would it have in handling the Apple?

Reply:
Crunchy vs pithy, and could end up in the rotten category?

Reply:
Ah got you. As a farmer marketing your Apple objects, would you consider any texture other than standard/edible to be worth pointing out separately from cotton or non-viable?